Dockerfile.ci

```diff
@@ -0,0 +1,36 @@
+# This Dockerfile is meant to be used in CI, as part of the release pipeline.
+# The goal is to ensure that the exact Python package that was built and released
+# is the one that is installed while creating this image. Prerequisites are:
+#   * The package has already been built (e.g., `poetry build -vvv`)
+#   * There is only one wheel built and put in the `dist` directory
```

Comment:
I figured out how to use the `ARG` directive in order to eliminate this file in favor of a single Dockerfile. Update coming...